filebeat/module/icinga/main/test/test.log-expected.json

```diff
@@ -4,32 +4,32 @@
         "event.dataset": "main", 
         "event.module": "icinga", 
         "icinga.main.facility": "Notification", 
-        "icinga.main.message": "Sending 'Recovery' notification 'demo!load!mail-icingaadmin for user 'on-call'", 
-        "icinga.main.severity": "information", 
         "input.type": "log", 
-        "log.offset": 0
+        "log.level": "information", 
```

Comment:
we should probably also standardise these in ECS so it's always "INFO". Nothing todo here, just a general comment.